GoogleForCreators/web-stories-wp
# Workspace: Update disclosure menu components

## Context

All menu-like components that use the chevron icon should have the same appearance and behavior, and use the same component where possible.

## Summary

Update instances of the 'chevron' icon in clickable elements used to show/hide content or menus with the `Disclosure` component and resolve related styling issues.

## Relevant Technical Choices

Refactored disclosure-type UI elements to use the `Disclosure` component for consistent appearance and implementation.

## To-do

- Address any unresolved styling issues
- Replace other disclosure-type UI elements as needed

## User-facing changes

Before - icons do not properly reflect open/closed state

<img width="298" alt="Screen Shot 2022-06-27 at 8 32 05 PM" src="https://user-images.githubusercontent.com/4173034/176068392-d0c6c48b-069e-46ea-a1ff-54e40103d83f.png">

<img width="197" alt="Screen Shot 2022-06-27 at 8 33 04 PM" src="https://user-images.githubusercontent.com/4173034/176068404-dd9253b7-e5bf-44b6-b7f0-634090cb946a.png">

<img width="224" alt="Screen Shot 2022-06-27 at 8 34 14 PM" src="https://user-images.githubusercontent.com/4173034/176068415-a0d99408-7d36-472a-ba7f-35b04238996d.png">

After - icons properly reflect open/closed state

<img width="291" alt="Screen Shot 2022-06-27 at 8 34 48 PM" src="https://user-images.githubusercontent.com/4173034/176068497-cd035749-56d3-480e-b464-c3bd4e3b48bb.png">

<img width="194" alt="Screen Shot 2022-06-27 at 8 35 16 PM" src="https://user-images.githubusercontent.com/4173034/176068509-aad9c666-f0d3-4378-a6eb-609eef8a7d6f.png">

<img width="198" alt="Screen Shot 2022-06-27 at 8 36 05 PM" src="https://user-images.githubusercontent.com/4173034/176068525-6d39ebae-a7c3-445d-a841-710372233135.png">


## Testing Instructions

Ensure 'disclosure' chevron icons rotate to reflect the open/closed state properly.




## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist




---



Fixes #11770 

NOTE: #11737 must be merged first, with this immediately following
